src/backend/src/modules/gcnotify/ro/gcnotify.ro.ts

```diff
@@ -0,0 +1,23 @@
+export class GcNotifyRO {
+  data: {
+    id: number;
+
+    reference: number | null;
+
+    to: string | number;
+
+    status: string;
+
+    created_at: Date;
+
+    completed_at: Date | null;
+
+    sent_at: Date | null;
+
+    notification_type: string;
+
+    template_id: number;
+
+    template_version: number;
+  };
```

Comment:
my bad - I misunderstood this last time. I thought the RO only returns the status. 

With all these info returned, unsure of the use case, but do we need to send all this data to Frontend?

Reply:
My main comment initially was to get rid of "any" as return type. If it has to be just the "string" status. That's also fine. Sorry for miscommunication, if any.

Reply:
Originally, my thought was that we only really want to know the status of the email, i.e.: delivered, permanent-failure, temporary-failure or technical-failure. Would you like me to change it to return `Promise<string>`?